modules/340-monitoring-kubernetes/monitoring/prometheus-rules/kubernetes/node.yaml

```diff
@@ -13,22 +13,15 @@
       plk_protocol_version: "1"
       plk_markup_format: markdown
       description: |
-        1. Please, check the NTP daemon's status by executing the following commands on the Node:
-           * for `ntpd` - 'ntpq -p' or 'ntpdc -c sysinfo' or 'ntpdc -c sysstats';
-           * for `chronyd` - 'chronyc sources -v' or 'chronyc tracking' or 'chronyc sourcestats -v' or 'chronyc ntpdata';
-           * for `systemd-timesyncd` - 'timedatectl status' or 'systemctl status systemd-timesyncd'.
-        2. Correct the time synchronization problems:
-           * restart NTP daemon:
-             - for `ntpd` - 'service ntp restart' or 'service ntpd restart' or 'systemctl restart ntp' or 'systemctl restart ntpd';
-             - for `chronyd` - 'systemctl restart chronyd';
-             - for `systemd-timesyncd` - 'systemctl restart systemd-timesyncd'.
+        1. Get Chrony pod on node:
+           * get pod name from 'kubectl -n d8-chrony --field-selector spec.nodeName="{{$labels.node}}" get pods'
+        2. check the Chrony daemon's status by executing the following command:
+           * 'kubectl -n d8-chrony exec <POD_NAME> -- /usr/bin/chronyc sources'
+        3. Correct the time synchronization problems:
            * correct network problems:
-             - provide availability to upstream time synchronization servers defined in the NTP daemon configuration file;
+             - provide availability to upstream time synchronization servers defined in [CM Deckhouse](https://deckhouse.io/en/documentation/v1/modules/470-chrony/configuration.html);
```

Comment:
If we want to alert only about the `chrony` errors, do we need to move these alerts to the `chrony` module directory then?

Reply:
Moved conf to chrony module